doc/guides/porting/board_porting.rst

```diff
@@ -120,6 +120,16 @@ Board
 A board implements an SoC with all its features, together with peripherals
 available on the board that differentiates the board with additional interfaces
 and features not available in the SoC.
+A board port on Zephyr typically consists of two parts:
```

Comment:
This feels like the beginning of a new paragraph, so add a blank line before it.